master/job.go

```diff
@@ -10,8 +10,14 @@ import (
 
 type job struct {
 	lib.Job
-	id    int
-	tasks []*lib.Task
+	id           int
+	tasks        []*lib.Task
+	runningTasks int
```

Comment:
runningTasks is superfluous, this can be calculated at runtime. Otherwise you could have a inconsistent struct.